lib/config/rules/typescript.js

```diff
@@ -0,0 +1,41 @@
+// Typescript Specific:
+// These rules find errors related to TypeScript features
+// Reference https://palantir.github.io/tslint/rules/
+
+module.exports = {
+  // Enforces function overloads to be consecutive.
+  'adjacent-overload-signatures': 'off',
```

Comment:
Why are these suddenly using string values instead of the `true`/ `false`? Also, we definitely want this one on.